### DISCUSSION TITLE
RustDesk fails to connect with red dot.

### DISCUSSION THREAD

**Discussion by macccorehenni** [rustdesk/rustdesk#5855] [Q&A]
I'm having issues with connecting to my father's computer via RustDesk. 
System version:
Father: Windows 11 build 22621.2361 (22H2)
Mine: on both Ubuntu 20.04 (KDE neon with Plasma 22.6) and Debian buster (Google Crostini with Google Garcon).
Rustdesk ver. for Father and I: 1.2.1 free
Hardware on my end: Lenovo Chromebook and another device (will edit later with exact computer name)
Hardware on my father's end: HP Laptop 14DQ1XXX
Expected result:
I am able to connect to Windows 11 as a guest.
Actual result:
I see a red dot with the message "Not ready. Please check your connection.". To be completely clear, I am able to reach the internet, and connection is otherwise able to be used.
How would I fix this? This is hindering my ability to connect to my father's computer for remote support.

**Comment by macccorehenni** [comment]
@jayreape-1 is Father.

**Reply by Mauri740** [reply]
I have the same problem. When attempting to connect I got a red dot with the error "Not ready. Please check your connection". This problem only occurs on one of my three PCs. Any ideas to solve? Thank you

**Reply by Netmistro** [reply]
I am also getting this error

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/5904
